src/opencensus/proto/error/v1/error.proto

```diff
@@ -0,0 +1,245 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opencensus.proto.error.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.proto.error.v1";
+option java_outer_classname = "ErrorProto";
+
+option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/error/v1";
+
+// Error event that is generated by integration or application.
+message Error {
+  // Time when the error occurred.
+  // This field is required.  
+  google.protobuf.Timestamp time = 1;
+
+  // Error description such as exception message.
+  // This field is required.  
+  string message = 2;
+
+  // TraceId of the span in which scope the error occurred. 
+  // Must be set if there is a current span or it was explicitly
+  // provided by the caller
+  bytes trace_id = 3;
```

Comment:
Should we add a SpanContext message? Also should we include the Tracestate?

Reply:
`SpanContext` sounds reasonable. And it will be good to have. One issue with this is inconsistency with spans. In span - there is a `parent_span_id` defined as a separate field. Also for logs the current thinking is separate fields (my understanding): https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/LogCorrelation.md#log-correlation